simpleCalculator.py

```diff
@@ -1,7 +1,9 @@
 import math
 
 #functions below
-
+def add(a,b):
+     result = a +b
+     return result 
```

Comment:
Could you change this to something like this "{0} + {1} = {2}".format(a, b, result)
Just to ensure uniformity in the return statement for all functions.